### DISCUSSION TITLE
发送涩图时只发送链接不发图片

### DISCUSSION THREAD

**Discussion by FlowerHZR** [Tsuk1ko/CQ-picfinder-robot#266] [问题求助] [answered]
![屏幕截图 2021-12-04 155822](https://user-images.githubusercontent.com/57387012/144702318-f959544b-5d80-4893-a424-a3f12d9ef0ae.png
![屏幕截图 2021-12-04 160103](https://user-images.githubusercontent.com/57387012/144702362-b13cea23-817c-41b5-8389-79e672063244.png)
)请问如何能发送图片

**Comment by Tsuk1ko** [comment, marked as answer]
可能是网络问题，建议配置代理，或者配置可访问的[在线反代](https://github.com/Tsuk1ko/cq-picsearcher-bot/wiki/%E9%99%84%E5%8A%A0%E5%8A%9F%E8%83%BD#p%E7%AB%99%E5%9B%BE%E7%89%87%E5%9C%A8%E7%BA%BF%E5%8F%8D%E4%BB%A3)

**Reply by FlowerHZR** [reply]
已解决，感谢
